client/my-sites/plans/jetpack-plans/jetpack-social-free-card/index.tsx

```diff
@@ -0,0 +1,88 @@
+import { recordTracksEvent } from '@automattic/calypso-analytics';
+import { TERM_ANNUALLY } from '@automattic/calypso-products';
+import classNames from 'classnames';
+import { useTranslate } from 'i18n-calypso';
+import { useCallback, useMemo } from 'react';
+import { useDispatch } from 'react-redux';
+import JetpackProductCard from 'calypso/components/jetpack/card/jetpack-product-card';
+import isJetpackCloud from 'calypso/lib/jetpack/is-jetpack-cloud';
+import { ITEM_TYPE_PRODUCT } from 'calypso/my-sites/plans/jetpack-plans/constants';
+import type { QueryArgs, SelectorProduct } from 'calypso/my-sites/plans/jetpack-plans/types';
+
+const SOCIAL_FREE_URL = isJetpackCloud()
```

Comment:
We'll want to come back and update this so the path is appropriate for each environment, rather than pointing at the `cloud.jetpack.com` domain for all Jetpack Cloud instances. For example, local testing and Calypso Live will have a different root domain.

Reply:
Yes, good point, thank you! I made it a relative path now.